publisher-sdk/src/main/java/com/criteo/publisher/logging/NativeLogMessage.kt

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
This comment applies to all classes that create `LogMessage` objects. I think these classes should not sit within the `logging` package. Rather, they should be in their respective feature package. `logging` package should be core only (something that can be extracted in a library for eg).

Reply:
Ok and this prevents the need of Kotlin extensions to make fields available internally (like https://github.com/criteo/android-publisher-sdk/blob/a6b3f90e733c11cf6ccc1381ac4e27abbf19482f/publisher-sdk/src/main/java/com/criteo/publisher/StandaloneExt.kt)